dotCMS/src/main/java/com/dotcms/graphql/business/ContentAPIGraphQLTypesProvider.java

```diff
@@ -134,6 +135,15 @@ private void fillTypesMap() throws DotDataException {
         List<ContentType> allTypes = APILocator.getContentTypeAPI(APILocator.systemUser())
                 .search("", null, 100000, 0);
 
+        // let's log if we are including dupe types
+        final Map<String, ContentType> localTypesMap = new HashMap<>();
+        allTypes.forEach((type)-> {
+            if(localTypesMap.containsKey(type.variable())) {
+                Logger.warn(this, "Dupe Content Type detected!: " + type.variable());
+            }
+            localTypesMap.put(type.variable(), type);
```

Comment:
what do you do with this map at the end? The collection is never used outside the foreach

Reply:
I got it, it is just used to find duplicates. NVM